sdk/keyvault/keyvault-certificates/src/transformations.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+import { CertificateOperationError } from ".";
```

Comment:
Won't this cause a recursive issue? 🤔

Reply:
It shouldn't since the barrel file exports it. But you're right that we should import it from the models file instead. Will fix!